src/Sql/dbo/Stored Procedures/UserSignatureKeyPair_UpdateForRotation.sql

```diff
@@ -0,0 +1,13 @@
+CREATE PROCEDURE [dbo].[UserSignatureKeyPair_SetForRotation]
```

Comment:
The name of this file and the contents of this file don't match. 

`[dbo].[UserSignatureKeyPair_SetForRotation]` code
`src/Sql/dbo/Stored Procedures/UserSignatureKeyPair_UpdateForRotation.sql` file name.

Reply:
Fixed